apps/sms/services/js/message_manager.js

```diff
@@ -137,28 +187,39 @@ var MessageManager = {
     };
   },
 
+  /**
+   * Function to get the message object having a given id.
```

Comment:
If you read MDN, you will find what it returns. So here you can add @return too in comments. Same at other places